core/types/src/state.rs

```diff
@@ -191,6 +192,8 @@ pub enum ProtocolParamKey {
     SGXSharedPubKey = 13,
     /// The number of epochs per year
     EpochsPerYear = 14,
+    /// The ping timeout for node reputation.
+    ReputationPingTimeout = 15,
```

Comment:
Is a protocol parameter right for this, or should it just be in the pinger config? It seemed like something that all participants should conform on, so I went with protocol param, but wanted to confirm.

Reply:
I think it makes sense, there are a few instances throughout the codebase where we define a const, but a protocol param would be better.